tests/unit/test_noise_form.py

```diff
@@ -129,7 +129,8 @@ def test_noise_order(mocker, dummy_data, dummy_config_noise_numbers):
 
     call_order = [x[0] for x in mock.mock_calls if not x[0].startswith("__")]
     expected_call_order = [
-        "omission",
+        # "omit_rows",   # Census doesn't use omit_rows
```

Comment:
Can you add a FIXME to update this to another dataset that does have `omit_rows`?

Reply:
You already have a FIXME to do something other than census. Do you have a ticket for that? Otherwise, we ought to create one.

Reply:
It didn't look like one existed, so I created [MIC-4037](https://jira.ihme.washington.edu/browse/MIC-4037).